src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -3,17 +3,57 @@
 import core.basesyntax.Storage;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    private static final int MAX_CAPACITY = 10;
+    private static final int NOT_FOUND_INDEX = -1;
+    private int size;
+    private K[] keys;
+    private V[] values;
+
+    public StorageImpl() {
+        this.keys = (K[]) new Object[MAX_CAPACITY];
+        this.values = (V[]) new Object[MAX_CAPACITY];
+    }
```

Comment:
```suggestion
        keys = (K[]) new Object[MAX_CAPACITY];
        values = (V[]) new Object[MAX_CAPACITY];
    }  
```
remove redundant this everywhere